examples/events/EiffelServiceStartedEvent/simple.json

```diff
@@ -0,0 +1,17 @@
+{
+  "meta": {
+    "type": "EiffelServiceStartedEvent",
+    "version": "1.0.0",
+    "time": 1234567890,
+    "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee5",
+    "source": {
+    "domainId": "example.domain"
+    }
+  },
+    "links": [
+      {
+      "type": "ENVIRONMENT",
+      "target": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee0"
+      }
```

Comment:
Yes , quite right,  
How would I acheive this .
Would this be acheived by using a links Object to the artifact deployment?
Or would this best acheived in a data Object ?
Or both ?